src/Symfony/Bundle/FrameworkBundle/Resources/config/amqp.xml

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" ?>
+
+<container xmlns="http://symfony.com/schema/dic/services"
+           xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+           xsi:schemaLocation="http://symfony.com/schema/dic/services http://symfony.com/schema/dic/services/services-1.0.xsd">
+
+    <services>
+        <defaults public="false" />
+
+        <service id="amqp.command.move" class="Symfony\Component\Amqp\Command\AmqpMoveCommand">
+            <argument type="service" id="amqp.broker" />
```

Comment:
You should use a connection locator here I guess, in case there are multiple connections defined.